convert2rhel/main.py

```diff
@@ -185,6 +189,11 @@ def _handle_main_exceptions(process_phase, pre_conversion_results=None):
     if process_phase == ConversionPhase.POST_CLI:
         loggerinst.info(no_changes_msg)
     elif process_phase == ConversionPhase.PRE_PONR_CHANGES:
+        # Update RHSM custom facts only when thi returns False. Otherwise,
+        # sub-man get uninstalled and the data is removed from the server.
```

Comment:
```suggestion
        # Update RHSM custom facts only when this returns False. Otherwise,
        # sub-man gets uninstalled and the data is removed from the server.
```